src/tuning/tune_hyperparameters.py

```
@@ -0,0 +1,361 @@
+        }
+
+        logger.info(
+            f"Trial completed - Train Loss: {trial_metrics['best_val_loss']:.4f} "
+            f"({eval_set_name} eval), "
+            f"Directional Acc: {trial_metrics['val_directional_accuracy']:.1f}%, "
+            f"RMSE: {trial_metrics['val_rmse']:.4f}, R²: {trial_metrics['val_r_squared']:.3f}"
+        )
+
+        return trial_metrics
+
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)